src/ui/Analytics/LiveAnalyticsClient.ts

```diff
@@ -277,7 +277,7 @@ export class LiveAnalyticsClient implements IAnalyticsClient {
       originContext: this.originContext,
       customData: _.keys(metaObject).length > 0 ? metaObject : undefined,
       userAgent: navigator.userAgent,
-      clientId: new AnalyticsInformation().clientId
+      clientId: new AnalyticsInformation().visitorId
```

Comment:
Why should the visitorId be mapped to the clientId? Doesn't this shift control of the id from the client back to server?

Reply:
I've added a thread on slack to discuss this.